middleware/routeHandler.js

```diff
@@ -1,10 +1,28 @@
+const { getCommitAndTreeSha, revertCommit } = require('../utils/utils.js')
+
 const attachRouteHandlerWrapper = (routeHandler) => async (req, res, next) => {
-    routeHandler(req, res).catch((err) => {
-      next(err)
-    })
+  routeHandler(req, res).catch((err) => {
+    next(err)
+  })
+}
+
+const attachRollbackRouteHandlerWrapper = (routeHandler) => async (req, res, next) => {
+  const { accessToken } = req
+  const { siteName } = req.params
+  let originalCommitSha
+  try {
+    const { currentCommitSha } = await getCommitAndTreeSha(siteName, accessToken)
+    originalCommitSha = currentCommitSha
+  } catch (err) {
+    next(err)
```

Comment:
Currently no error will be caught because `getCommitAndTreeSha` does not throw any errors, it simply console.logs errors

Reply:
Right, changed to throw error instead in https://github.com/isomerpages/isomercms-backend/pull/91/commits/16362d1bc82615e3eac2e44234650a11d52ccbc5